crates/rpc/src/v05/method/simulate_transactions.rs

```diff
@@ -431,12 +431,14 @@ pub mod dto {
     pub struct L1HandlerTxnTrace {
         #[serde(default)]
         pub function_invocation: Option<FunctionInvocation>,
+        pub state_diff: StateDiff,
```

Comment:
Should this not be optional?

Reply:
It should -- I'll update that in a follow-up PR.